src/components/scene/canvas.js

```diff
@@ -28,6 +28,8 @@ module.exports.Component = register('canvas', {
       canvas.classList.add('a-canvas');
       canvas.style.height = data.height + '%';
       canvas.style.width = data.width + '%';
+      // Mark canvas as provided/injected by A-Frame.
+      canvas.dataset.aframeProvided = true;
```

Comment:
In the light and camera systems we mark the default elements like this:
````javascript
ambientLight.setAttribute('data-aframe-default-light', '');
cameraWrapperEl.setAttribute('data-aframe-default-camera', '');
````
I think we should either change this or change the camera and light systems to follow this convention. @ngokevin what do you think?